<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerHtmlWidgetBase</class>
 <widget class="QWidget" name="QgsComposerHtmlWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QToolBox" name="mToolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>100</height>
       </rect>
      </property>
      <attribute name="label">
       <string>HTML</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QToolButton" name="mFileToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mUrlLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mUrlLabel">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="mResizeModeComboBox"/>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="mResizeModeLabel">
         <property name="text">
          <string>Resize mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
